zebra-consensus/Cargo.toml

```diff
@@ -51,6 +51,6 @@ tokio = { version = "1.13.0", features = ["full"] }
 tracing-error = "0.1.2"
 tracing-subscriber = "0.2.25"
 
-zebra-chain = { path = "../zebra-chain", features = ["proptest-impl"] }
+zebra-chain = { path = "../zebra-chain", features = ["proptest-impl", "bench"] }
```

Comment:
Question: I'm curious as to why the extra feature dependency was added :sweat_smile:

Reply:
`zebra_chain::block::test::generate` is feature-gated with `bench`. But I needed to use a large generated transaction to check that the sigops limit was actually enforced.

But I think it's less confusing to expose that module under `proptest-impl` as well. (Which we will rename in #2424.)

Fixed in 76f3d383c